content/news/XXXX-XX-XX-mixxx-2_5-released.md

```diff
@@ -0,0 +1,445 @@
+title: Mixxx 2.5 Release Announcement

```

Comment:
Maybe more clear that it's already available when users read this text :-) 

```suggestion
title: Mixxx 2.5 Released
```

Reply:
I think @JoergAtGithub prefers these tags to be uniform.

Reply:
This the title, not the tag. If you click on the "release announcement" tag you see all the posts which have that tag.

Reply:
owkee